scripts/backup

```diff
@@ -43,14 +44,16 @@ upstream_version=$(ynh_app_upstream_version)
 #=================================================
 ynh_script_progression --message="Backing up the main app directory..." --weight=1
 
-ynh_backup --src_path="$final_path"
```

Comment:
Why did you remove this line? Is it intended?

Reply:
No I guess it's a mistake...
Josue-T asked me to drop one line below this but I didn't want to remove that at the same time.